constraint/pkg/apis/templates/v1/helpers.go

```diff
@@ -0,0 +1,30 @@
+package v1
+
+import (
+	"fmt"
+
+	"github.com/open-policy-agent/frameworks/constraint/pkg/core/templates"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+// ToVersionless runs defaulting functions and then converts the ConstraintTemplate to the
+// versionless api representation
+func (versioned *ConstraintTemplate) ToVersionless(scheme *runtime.Scheme) (*templates.ConstraintTemplate, error) {
```

Comment:
We shouldn't require users to pass a scheme, just make your own local scheme for the frameworks package, I think we already have one:

https://github.com/open-policy-agent/frameworks/blob/c000ec35322ee04a6bc0a04041416432dfea943c/constraint/pkg/client/crd_helpers.go#L83-L89

Reply:
done